jormungandr-lib/src/interfaces/block0_configuration/initial_fragment.rs

```diff
@@ -72,6 +72,13 @@ pub fn try_initials_vec_from_messages<'a>(
                 let cert = certificate::SignedCertificate::StakeDelegation(cert, auth);
                 inits.push(Initial::Cert(cert.into()))
             }
+            Fragment::VotePlan(tx) => {
+                let tx = tx.as_slice();
+                let cert = tx.payload().into_payload();
+                let auth = tx.payload_auth().into_payload_auth();
+                let cert = certificate::SignedCertificate::VotePlan(cert, auth);
```

Comment:
the auth here will be empty, right?

Reply:
yes. it "future" proof it so if we go for something else we can still change our mind :)